packages/admin/cms-admin/src/pages/pageTree/PageTree.tsx

```diff
@@ -70,7 +70,7 @@ const MOVE_PAGE_TREE_NODES_BY_NEIGHBOURS = gql`
     }
 `;
 
-const RESET_SLUG = gql`
+const restSlugMutation = gql`
```

Comment:
should be resetSlugMutation I suppose...

Reply:
uups

Reply:
fixed spelling issue.